pkg/executor/aggregate/agg_hash_executor.go

```diff
@@ -316,34 +318,78 @@ func (e *HashAggExec) initForParallelExec(_ sessionctx.Context) {
 		e.inputCh <- input
 	}
 
-	// Init final workers.
+	runningWorkerWaiter.Add(partialConcurrency)
```

Comment:
Looks like it is only used to wait the running partial worker?

Reply:
> Looks like it is only used to wait the running partial worker?

Yes it's only used for waiting all running partial workers.

Reply:
> Then how about rename it to `runningPartialWorkerWaiter`?

okk